proposals/XXXX-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -132,7 +132,7 @@ func postUrl(@Lowercased urlString: String) {
 postUrl(urlString: "mySite.xyz/myUnformattedUsErNAme")
 ```
 
-What's more, if `Binding` conditionally conformed to `RandomAccessCollection`, property-wrapper parameters could be used with `ForEach` to access collection elements directly in the `content` closure and enable property-wrapper syntax in the closure body:
+What's more, if `Binding` conditionally conformed to `RandomAccessCollection`, property-wrapper parameters could be used with `ForEach` to access collection elements directly in the `content` closure; thus enabling property-wrapper syntax in the closure body:
```

Comment:
"thus enabling" doesn't make sense here; "accessing collection elements directly" and "enabling property-wrapper syntax" are two separate benefits. The first one does not imply the second one.